docs-web/configuration/security.md

```diff
@@ -0,0 +1,42 @@
+# Security
+
+We consider the security of the Ingress Controller paramount to the success of our users and use the following to 
+ensure the security of the Ingress Controller:
+* [Secure Development Life Cycle](https://www.microsoft.com/en-us/securityengineering/sdl/)
+* [FOSSA](https://fossa.com) scanning
+
+However, the Ingress Controller is deployed by a User in their environment, and as such, the User takes responsibility 
+for securing a *deployment* of the Ingress Controller. 
+We highly recommend every User to read and understand the following security concerns.
+
+## Kubernetes
+We recommend the Kubernetes [guide to securing a cluster](https://kubernetes.io/docs/tasks/administer-cluster/securing-a-cluster/).
+In addition, the following relating more specifically to Ingress Controller.
+
+### RBAC and Service Account
+The Ingress Controller is deployed within a Kubernetes environment, this environment must be secured. 
+Kubernetes uses [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to control which types of users 
+can access which resources and what operations they can perform. 
+The Ingress Controller requires a service account which is configured using RBAC. 
+We strongly recommend using the [RBAC configuration](https://github.com/nginxinc/kubernetes-ingress/blob/master/deployments/rbac/rbac.yaml) provided in our standard deployment configuration. 
+It is configured with the least amount of privilege required for the Ingress Controller to work.
+
+### Certificates and Privacy Keys
```

Comment:
Certificates are public, but private keys - not
```suggestion
### Private Keys
```

Reply:
https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets

This section refers to Kubernetes Secrets. In this case, TLS secrets, which has a cat and key value. I mention TLS specifically as we use it for a few cli arguments

Reply:
I see. Would it make sense to name the section "Secrets" then? Note that the IC supports the following secrets:
- kubernetes.io/tls for TLS secrets.
- nginx.org/jwk for JWK secrets.
- nginx.org/ca for CA secrets.
- nginx.org/oidc - for OIDC client secret value. 

So not every Secret has certificate and private key. 
Also note that it is private, not privacy key.